behandlingslager/domene/src/main/java/no/nav/foreldrepenger/behandlingslager/behandling/tilrettelegging/SvpAvklartOpphold.java

```diff
@@ -0,0 +1,127 @@
+package no.nav.foreldrepenger.behandlingslager.behandling.tilrettelegging;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import no.nav.foreldrepenger.behandlingslager.BaseEntitet;
+import no.nav.foreldrepenger.behandlingslager.diff.IndexKey;
+
+
+@Entity
+@Table(name = "SVP_AVKLART_OPPHOLD")
+public class SvpAvklartOpphold extends BaseEntitet implements IndexKey {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "SEQ_SVP_AVKLART_OPPHOLD")
+    private Long id;
+
+    @Column(name = "fom", nullable = false)
```

Comment:
Her kunne vi bare gjenbrukt `DatoIntervallEntitet` som definerer for og tom kolonner.

Reply:
Jeg endrer